### DISCUSSION TITLE
Feature Request: Zoom out a single level

### DISCUSSION THREAD

**Discussion by MattTheVV** [vslinko/obsidian-zoom#109] [Ideas]
This plug-in is quite amazing for focusing on parts of a note, much appreciated!

What I'm missing, however, is the ability to zoom out a single level.
(currently we only have two keybindings: Zoom In and Zoom Out to Page)

# Example

```markdown
### Header Level 1
Text 1

#### Header Level 2
- Bullet 1
```
#

### Desired Behavior

If I zoom into Header Level 2, I'd like to be able move up a level to Header Level 1 with a keybinding (e.g. `CTRL+,`): instead of zooming out to the page.

**Comment by freefaler** [comment]
This indeed can be beneficial when working with large outlines.
